benchmarks/parquet.rs

```diff
@@ -178,6 +299,22 @@ async fn exec_scan(
     Ok(result.iter().map(|b| b.num_rows()).sum())
 }
 
+async fn exec_sort(
+    ctx: &SessionContext,
+    expr: &[PhysicalSortExpr],
+    test_file: &TestParquetFile,
+    debug: bool,
+) -> Result<()> {
+    let scan = test_file.create_scan(lit(true)).await?;
```

Comment:
I am not sure the `lit(true)` filter will be removed at this level. What do you think about making `TestParquetFile::create_scan` take `Option<Expr>` instead of passing in lit true?

Reply:
Yeah that is a good solution. My initial thinking was I didn't want to change another crate (`parquet-test-utils`).

Reply:
It seem's like this is the only place in the codebase that is using this though, so it should be fine to change that method.